### PR TITLE
LPS-136681 Retrieve objects' field groups entries correctly

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-136681

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java [hunk 1]
```diff
@@ -153,51 +154,77 @@ public DDMStorageAdapterSaveResponse save(
...
+				DDMFormFieldValue ddmFormFieldValue = new DDMFormFieldValue();
 
-			ddmFormValues.addDDMFormFieldValue(ddmFormFieldValue);
-		}
+				ddmFormFieldValue.setName(ddmFormField.getName());
+
+				Value value = _getDDMFormFieldValue(
+					ddmFormFieldsMap, defaultLocale, properties, ddmFormField);
+
+				ddmFormFieldValue.setValue(value);
```
Comment (gabrielwas) [inline modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:209]: @mateussgn can we inline the `value` variable?

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java [hunk 2]
```diff
@@ -153,51 +154,77 @@ public DDMStorageAdapterSaveResponse save(
 		}
 	}
 
-	private DDMFormValues _getDDMFormValues(
-		DDMForm ddmForm, ObjectEntry objectEntry) {
+	private Value _getDDMFormFieldValue(
+		Map<String, DDMFormField> ddmFormFieldsMap, Locale defaultLocale,
+		Map<String, Object> properties, DDMFormField ddmFormField) {
```
Comment (gabrielwas) [inline modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:159]: @mateussgn can we sort this?

**Comment by gabrielwas** [comment]
Just started reviewing :)

**Comment by gabrielwas** [comment]
@mateussgn btw, good pull!

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java (hunk 1 above)
Comment (mateussgn) [reply modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:209]: Sure!
Thanks for the tip!

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java (hunk 2 above)
Comment (mateussgn) [reply modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:159]: Yes, we can 👍🏼

**Comment by mateussgn** [comment]
ci:forward

**Pull request closed without merging**